lecturer/junit_intro/src/main/java/by/jrr/start/Calculator.java

```diff
@@ -4,12 +4,12 @@
 
     private int result;
 
-    public int sum(int a, int b) {
-        return a + b;
+    public int multiplikation(int a, int b) {
+        return a * b;
     }
 
-    public int subs(int a, int b) {
-        return a - b;
+    public int division(int a, int b) {
```

Comment:
should perform only in b != 0;

Reply:
I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files